modules/saml/lib/SP/LogoutStore.php

```diff
@@ -40,7 +40,7 @@ private static function createLogoutTable(Store\SQL $store): void
                 case 'pgsql':
                     // This does not affect the NOT NULL constraint
                     $update = [
-                        'ALTER TABLE ' . $store->prefix . '_saml_LogoutStore ALTER COLUMN _expire TIMESTAMP'
+                        'ALTER TABLE ' . $store->prefix . '_saml_LogoutStore ALTER COLUMN type _expire TIMESTAMP'
```

Comment:
Shouldn't it be the other way around? `ALTER COLUMN _expire TYPE TIMESTAMP`?

Reply:
Yes, it looks better.